### ISSUE TITLE
Script failure after some time

### ISSUE THREAD

**Issue by piebaker** [RomeoCantCode/PSNorthstarWatcher#13]
Unsure of the conditions for this error, just noticed it. Server had been running for ~2hrs empty when this error occurred:
```
Get-NetUDPEndpoint : The remote procedure call failed.
At D:\Program Files (x86)\Origin Games\TF2\northstar server watcher.ps1:249 char:21
+     $localportudp = Get-NetUDPEndpoint | Where-Object -Property local ...
+                     ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (MSFT_NetUDPEndpoint:ROOT/StandardCimv2/MSFT_NetUDPEndpoint) [Get-NetUD
   PEndpoint], CimException
    + FullyQualifiedErrorId : HRESULT 0x800706be,Get-NetUDPEndpoint
```

**Comment by piebaker** [comment]
Seems the script is still running fine despite the error, strange.

**Comment by RomeoCantCode** [comment]
did the error ony appear once?

**Comment by piebaker** [comment]
Yup, just the once by the looks. Figured it was still worth reporting

**Comment by RomeoCantCode** [comment]
hey I will close this for now since I released v1.0 which takes a very different approach than the sketchy script.
